### PR TITLE
feat(ui): build personal information and register interface

### PR DESCRIPTION
**Pull request by ntu34543**
Resolves:
 [[BE] [Sign in]](https://trello.com/c/5NnWHMMb/27-be-sign-in)
 [[FE][Sign up] Build interface](https://trello.com/c/5tmLZBFP/42-fesign-up-build-interface)
 [[FE] [Influencer] Personal Information interface](https://trello.com/c/Ej6PsylS/45-fe-influencer-personal-information-interface)

### CODE REVIEW

**Comment by ntu34543** [comment]
[[FE][Sign up] Build interface](https://trello.com/c/5tmLZBFP/42-fesign-up-build-interface)

**Comment by ntu34543** [comment]
[[FE] [Influencer] Personal Information interface](https://trello.com/c/Ej6PsylS/45-fe-influencer-personal-information-interface)

File: package-lock.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "@fortawesome/free-brands-svg-icons": "^6.0.0",
```
Comment (uica-dev) [inline package-lock.json:12]: No need to push this lock file, they can install by running `npm install` or `yarn install` for themself.

File: src/view/pages/Influencers/PersonalInformation/Job.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Button from "../../../../components/Button/Button";
+import classNames from "classnames/bind";
+import styles from "./Job.module.scss";
+
+const cx = classNames.bind(styles);
```
Comment (uica-dev) [inline src/view/pages/Influencers/PersonalInformation/Job.js:6]: What does `cx` mean? Should be named clearly for others to easily understand!

**Review by uica-dev — APPROVED** [review]
I left some comments but the code LGTM btw! 🐱‍🏍

File: src/view/pages/Influencers/PersonalInformation/Job.js (hunk 2 above)
Comment (LazyyNguyen) [reply src/view/pages/Influencers/PersonalInformation/Job.js:6]: For easier reading when we use multiple variables for className.
Ex: <button className={`btn btn__summit ${isDangerField ? 'btn--danger' : ''} ${isRoundedField ? 'btn--rounder' : ''} ${isDisabled ? 'btn-disable' : ''}`}>Submit</button>
=> very messy
<button className={classnames(
       'btn',
       'btn__summit',
       { 'btn--danger': isDangerField },
       { 'btn--rounder': isRoundedField },
       { 'btn-disable': isDisabled }
     )}>Submit</button>
=> more clear

**Pull request merged**